data/json/items/containers.json

```diff
@@ -40,14 +40,27 @@
         "color": "light_cyan",
         "name": "plastic cup",
         "category": "other",
-        "description": "A small, vacuum formed cup, essentially trash.",
+        "description": "A small, vacuum formed cup.",
         "price": 0,
         "weight": 6,
         "volume": 1,
         "to_hit": 1,
         "material": "plastic",
         "contains": 1,
-        "watertight": true
+        "watertight": true,
+        "unseals_into": "cup_plastic_unsealed"
+    },
+    {
+        "id": "cup_plastic_unsealed",
+        "type": "CONTAINER",
+        "symbol": ")",
+        "color": "light_cyan",
+        "copy-from": "cup_plastic",
+        "name": "unsealed plastic cup",
+        "description": "A small, vacuum formed cup, essentially trash.",
+        "contains": 1,
```

Comment:
`contains` and `watertight` should not need specifying.
You should update `islot_container` to support inheritance as part of this PR.